content/nginx-one/api/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: API
+title: Automate with the NGINX One API
```

Comment:
"automate" is a transitive verb -- not having a direct object feels weird to me.

Reply:
I hear you. But I also hear discussions like: 

- Do you have to do this again and again? 
   - No, after the first time, I "automate"